### PR TITLE
 [CARBONDATA-1554] Add a new profile spark2.2 in main pom file

### PR DESCRIPTION
**Pull request by chenliang613**
Add a new profile spark2.2 in main pom file.

 - [X] Any interfaces changed?
    NA

 - [X] Any backward compatibility impacted?
    NA

 - [X] Document update required?
    NA

 - [X] Testing done
    NA
            
 - [X] For large changes, please consider breaking it into sub-tasks under an umbrella JIRA. 
    NA

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/734/

**Comment by ravipesala** [comment]
SDV Build Success , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/1371/

File: pom.xml [hunk 1]
```diff
@@ -515,6 +515,17 @@
       </build>
     </profile>
     <profile>
+      <id>spark-2.2</id>
+      <properties>
+        <spark.version>2.1.0</spark.version>
+        <scala.binary.version>2.11</scala.binary.version>
+        <scala.version>2.11.8</scala.version>
```
Comment (jackylk) [inline pom.xml]: Should not hard code the version

File: pom.xml [hunk 2]
```diff
@@ -515,6 +515,17 @@
       </build>
     </profile>
     <profile>
+      <id>spark-2.2</id>
+      <properties>
+        <spark.version>2.1.0</spark.version>
```
Comment (jackylk) [inline pom.xml]: It should be 2.2.0

Comment (chenliang613) [reply pom.xml]: ok

File: pom.xml (hunk 1 above)
Comment (chenliang613) [reply pom.xml]: this is not hard code. by default use spark 2.1.0, here should change to 2.2.0

File: pom.xml [hunk 3]
```diff
@@ -515,6 +516,51 @@
...
     <profile>
+      <id>spark-2.2</id>
+      <properties>
+        <spark.version>2.2.0</spark.version>
+        <scala.binary.version>2.11</scala.binary.version>
+        <scala.version>2.11.8</scala.version>
+      </properties>
+      <modules>
+        <module>integration/spark2</module>
+        <module>examples/spark2</module>
```
Comment (zzcclp) [inline pom.xml]: @chenliang613 I have raised a pr to add package integration/spark2.2 and examples/spark2.2, we should change these two lines to spark2.2?

**Comment by chenliang613** [comment]
already merged to branch spark2.2, so close this pr.

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/742/

**Comment by ravipesala** [comment]
SDV Build Success , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/1379/

**Pull request closed without merging**